{
  "comments": [
    {
      "key": {
        "uuid": "55a15ea3_3ae5aa84",
        "filename": "/COMMIT_MSG",
        "patchSetId": 3
      },
      "lineNbr": 32,
      "author": {
        "id": 1016895
      },
      "writtenOn": "2019-09-17T14:05:38Z",
      "side": 1,
      "message": "Probably need to add the following after an import:\n\n1. time sync force\n2. some sort of health check and fixup of well known issues after reboot, like cinder-volume not coming up due to the local loopback device not being in place",
      "revId": "909abbef395a93f0328280e48bced677878d8d76",
      "serverId": "d5d70762-12d0-45a1-890d-524b12d3f735",
      "unresolved": true
    },
    {
      "key": {
        "uuid": "01f31ecc_eb8bbfd7",
        "filename": "plugins/virsh/snapshot_import.yml",
        "patchSetId": 3
      },
      "lineNbr": 7,
      "author": {
        "id": 1016895
      },
      "writtenOn": "2019-09-17T14:03:26Z",
      "side": 1,
      "message": "This resulted in \u0027cloud-user\u0027 instead of \u0027stack\u0027 in a recent test. I need to track down why that is. I may perhaps be using the wrong variable here.",
      "range": {
        "startLine": 7,
        "startChar": 27,
        "endLine": 7,
        "endChar": 54
      },
      "revId": "909abbef395a93f0328280e48bced677878d8d76",
      "serverId": "d5d70762-12d0-45a1-890d-524b12d3f735",
      "unresolved": true
    }
  ]
}